src/v3/src/transformer/field/type.ts

```diff
@@ -39,6 +39,7 @@ const map = new Map<string, JsonSchemaType>([
   ['pdatetime', 'string'],
   ['ptime', 'string'],
   ['string', 'string'],
+  ['text', 'string'],
```

Comment:
Registration parseSchema can send `text` as the `Input` type, so we need to make sure this is rendered properly.